src/common/utilities/OrgHandlerUtils.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+import { ExtensionContext } from 'vscode';
+import { ActiveOrgOutput } from '../../client/pac/PacTypes';
+import { PacWrapper } from '../../client/pac/PacWrapper';
```

Comment:
at some point we should consider moving this logic to common - that will keep referencing flow sorted. BUT see if we can atleast still remove this reference in common module. Copilot should be the only exception till we figure long term arrangement.